backend/src/packages/article-reactions/article-reaction.model.ts

```diff
@@ -0,0 +1,49 @@
+import { Model, type RelationMappings } from 'objection';
+
+import {
+  AbstractModel,
+  DatabaseTableName,
+} from '~/libs/packages/database/database.js';
+import { composeDatabaseRelationPath } from '~/libs/packages/database/libs/helpers/helpers.js';
+
+import { ArticleModel } from '../articles/article.model.js';
+import { UserModel } from '../users/user.model.js';
+
+class ArticleReactionModel extends AbstractModel {
+  public 'userId': number;
+  public 'articleId': number;
+  public 'isLike': boolean;
+
+  public static override get tableName(): string {
+    return DatabaseTableName.ARTICLE_REACTIONS;
+  }
+
+  public static get relationMappings(): RelationMappings {
+    return {
+      article: {
+        relation: Model.HasOneRelation,
+        modelClass: ArticleModel,
+        join: {
+          from: composeDatabaseRelationPath<ArticleReactionModel>(
+            'article_reactions',
```

Comment:
Use DatabaseTableName enum - for all table names on relations

Reply:
These names are suggested by autocomplete, change anyway?

Reply:
Yes, use enum